### PR TITLE
rules for OCP-28954

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei please help to review thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.5/operator_hub.xyaml [hunk 1]
```diff
@@ -179,3 +179,15 @@ uninstall_operator_on_console:
   action: click_resource_action_icon
   action: click_button
   action: submit_changes
+check_default_configuration_view:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/operator_hub.xyaml:182]: `check_default_view_is_form_view`

File: lib/rules/web/admin_console/4.5/operator_hub.xyaml [hunk 2]
```diff
@@ -179,3 +179,15 @@ uninstall_operator_on_console:
...
+  params:
+    content: Create by completing the form.
+  action: check_page_contains
+  element:
+    selector:
+      xpath: //input[@type='radio' and @value='form' and @checked]
+configure_via_yaml_view:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/operator_hub.xyaml:189]: `switch/toggle_to_yaml_view`

File: lib/rules/web/admin_console/4.5/operator_hub.xyaml [hunk 3]
```diff
@@ -179,3 +179,15 @@ uninstall_operator_on_console:
   action: click_resource_action_icon
   action: click_button
   action: submit_changes
+check_default_configuration_view:
+  params:
+    content: Create by completing the form.
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/operator_hub.xyaml:184]: It would be more stable if we can remove the ending dot

**Comment by SSshahan** [comment]
@pruan-rht  @liangxia  please help to merge, thanks

File: lib/rules/web/admin_console/4.5/goto.xyaml [hunk 4]
```diff
@@ -192,6 +192,10 @@ goto_one_project_subscription_page:
 goto_csv_detail_page:
   url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~ClusterServiceVersion/<csv_name>
   action: wait_box_loaded
+goto_operand_list_page:
+  url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~ClusterServiceVersion/<csv_name>/<operand_name>
+  action: wait_box_loaded
+
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.5/goto.xyaml:198]: Extra empty line. Not a big deal, but something we need to take care.

**Pull request merged**